genvm/core/context.go

```diff
@@ -121,13 +126,13 @@ func (c *Context) Consumed() uint64 {
 
 // Fee computed from consumed gas.
 func (c *Context) Fee() uint64 {
-	return c.consumed * c.Header.GasPrice
+	return c.fee
 }
 
 // Updated list of addresses.
 func (c *Context) Updated() []types.Address {
-	rst := make([]types.Address, len(c.touched)+1)
+	rst := make([]types.Address, 0, len(c.touched)+1)
 	rst = append(rst, c.Account.Address)
-	copy(rst[1:], c.touched)
+	rst = append(rst, c.touched...)
```

Comment:
curious about why you made a impl change here? is it performance or memory related?

Reply:
neither) i just noticed that i made a mistake